RockLib.Messaging/SenderMessage.cs

```diff
@@ -126,7 +148,7 @@ public SenderMessage(IReceiverMessage receiverMessage, Func<object, object> vali
         public bool IsCompressed =>
             Headers.TryGetValue(HeaderNames.IsCompressedPayload, out var value)
                 && ((value is bool isCompressed && isCompressed)
-                    || value is string isCompressedString && isCompressedString.ToLowerInvariant() == "true");
+                    || value is string isCompressedString && isCompressedString.ToUpperInvariant() == "TRUE");
```

Comment:
You just want to really emphasize if it's true?:)

Reply:
Well....this is when an analyzer is telling me that using `ToUpperInvariant()` is the "better" approach [link](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1308).

Reply:
ah interesting.